lib/options.nix

```diff
@@ -280,7 +280,7 @@ rec {
               renderOptionValue opt.example
             );
         }
-        // optionalAttrs (opt ? default) {
+        // optionalAttrs (opt ? defaultText || opt ? default) {
```

Comment:
Please update the corresponding test, or add one.

Reply:
Looks like we're not even testing `optionAttrSetToDocList`? I don't feel like looking into this, sorry

Reply:
No wonder it was broken then.